src/api/java/com/minecolonies/api/crafting/ItemStorage.java

```diff
@@ -98,11 +99,17 @@ public ItemStorage(@NotNull final ItemStack stack)
     {
         this.stack = stack;
         this.shouldIgnoreDamageValue = false;
-        this.shouldIgnoreNBTValue = false;
+        this.shouldIgnoreNBTValue = getDefaultIgnoreNBT(stack);
         this.amount = ItemStackUtils.getSize(stack);
         this.creativeTabIndex = stack.getItem().getCreativeTabs().stream().filter(Objects::nonNull).map(g -> g.index).collect(Collectors.toList());
     }
 
+    private static boolean getDefaultIgnoreNBT(@NotNull ItemStack stack)
+    {
+        //noinspection ConstantConditions
+        return stack.hasTag() && stack.getTag().getBoolean(NbtTagConstants.TAG_IGNORE_NBT);
```

Comment:
You might want to consider making this a Tag.
Since then other mods can add the compatibility directly, incase they also have items with this behaviour.

Reply:
I'm not quite sure what you mean by this.  It is passed through the tag?

Reply:
Oh, you mean a datapack item tag?  That could be useful too, although it's a much bigger stick (this one affects only specific recipes, that would affect all recipes).